ci-operator/config/openshift/dynamic-plugin-sdk/openshift-dynamic-plugin-sdk-main.yaml

```
@@ -10,9 +10,10 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+test_binary_build_commands: ./build.sh
 tests:
 - as: test
```

```suggestion
- as: all-packages
```

Console has their CI tests grouped based on logical components (frontend, backend, e2e, etc.) - for now, I'd suggest to name our CI test `all-packages` to better reflect its purpose.